src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1098,6 +1098,9 @@ def update_vm(cmd, resource_group_name, vm_name, os_disk=None, disk_caching=None
         else:
             vm.billing_profile.max_price = max_price
 
+    if proximity_placement_group is not None:
+        vm.proximity_placement_group = {'id': proximity_placement_group}
```

Comment:
Is there a model for proximity_placement_group?

Reply:
in params.py:
with self.argument_context(scope, min_api='2018-04-01')